### PR TITLE
feat: bump grafana-loki-chart to 0.79.4

### PR DESCRIPTION
**Pull request by Venkat-Bramhasani**
**What problem does this PR solve?**:

bumping-grafana-loki-chart to 0.79.4

**Which issue(s) does this PR fix?**:
CVE fix for grafana-loki-chart

### CODE REVIEW

**Comment by Venkat-Bramhasani** [comment]
@mesosphere-ci rebase

**Comment by Venkat-Bramhasani** [comment]
Hi @mhrabovcin , could you please take a look at this PR? I’m trying to figure out how to trigger the "Go Test" job. I noticed that Build 11553315653 is closed, so it's not triggering. Thanks!

File: services/grafana-loki/0.79.4/grafana-loki-helmrelease/grafana-loki.yaml
Comment (mhrabovcin) [inline services/grafana-loki/0.79.4/grafana-loki-helmrelease/grafana-loki.yaml:1]: The service number directory needs to be updated.

**Comment by mhrabovcin** [comment]
@Venkat-Bramhasani if this happens the fix is usually to close and re-open the PR.

**Comment by Venkat-Bramhasani** [comment]
Hi @mhrabovcin , as you suggested i made required changes, could you please review and take the necessary action.

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**